documentation/book/assembly-kafka-cluster.adoc

```diff
@@ -0,0 +1,39 @@
+// This assembly is included in the following assemblies:
+//
+// getting-started.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+:parent-context: {ContextProduct}
+
+include::common/attributes.adoc[]
+
+[id='kafka-broker-{context}']
+= Kafka cluster
+
+When installing Kafka, {ProductName} also installs a Zookeeper cluster and adds the necessary configuration for the Kafka broker connection.
+
+{ProductName} uses `StatefulSets` to deploy Kafka brokers.
+With `StatefulSets`, the pods receive a unique name and network identity, making it easier to identify the
+individual Kafka broker pods and set their identity (broker ID). For more information about `StatefulSets`, see {StatefulSets}.
+
+{ProductName} provides two options for Kafka cluster deployment:
+
+Ephemeral:: is suitable only for development and testing purposes and not for production. This deployment uses `emptyDir` volumes for storing broker information (Zookeeper) and topics or partitions
+(Kafka). Using an `emptyDir` volume means that its content is strictly related to the pod life cycle and is
+deleted when the pod goes down. This makes the ephemeral deployment well-suited to development and testing because you do not have to provide persistent volumes.
```

Comment:
I'm not really sure it is well suited for development. I would remove the last sentence.